documentation/docs/Exercises/_category_.json

```
@@ -1,4 +1,4 @@
 {
-  "label": "Exercises",
+  "label": "Exercises (coming soon)",
   "position": 4
-}
+}
```

Missing newline

---

My IDE keeps stripping away the newline.  I'll add it via this github comment 😛 
```suggestion
}

```